apps/user-portal/src/api/multi-factor-totp.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AxiosHttpClient } from "@wso2is/http";
+import { GlobalConfig, ServiceResourcesEndpoint } from "../configs";
+import { HttpMethods } from "../models";
+
+/**
+ * Get an axios instance.
+ *
+ * @type {AxiosHttpClientInstance}
+ */
+const httpClient = AxiosHttpClient.getInstance();
+
+/**
+ * The action types of the totp post endpoint
+ */
+enum POST_TOTP {
```

Comment:
We have used pascal case plural for enums in other placed. May be we can change `POST_TOTP` to something line `PostTOTPActions` make it consistent.